src/components/plots/piePlot.ts

```diff
@@ -67,6 +79,82 @@ export module Plots {
 
       return allPlotData;
     }
+
+    public sectorValue(): AccessorScaleBinding<D, number>;
+    public sectorValue(sectorValue: number | _Accessor): Plots.Pie<D>;
```

Comment:
Yup, I'm planning on making `_Accessor`s generic.  That'll add a bit more generics to this file, but it'll make things a lot more precise.